ingest/pipeline/storage.py

```
@@ -0,0 +1,217 @@
+            )
+
+        if not nodes:
+            logger.warning("No vectors to upsert")
+            return 0
+
+        logger.info(f"Upserting {len(nodes)} vectors to Qdrant")
+
+        # ポイントの作成
+        points = []
+        for node, embedding in zip(nodes, embeddings, strict=True):
+            # 埋め込みが無効（すべてゼロ）の場合はスキップ
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)